client/package.json

```diff
@@ -3,11 +3,12 @@
     "license": "GPL-3.0",
     "dependencies": {
         "superdesk-analytics": "github:superdesk/superdesk-analytics#v2.8.0",
-        "superdesk-core": "github:superdesk/superdesk-client-core#v2.8.0",
-        "superdesk-planning": "github:superdesk/superdesk-planning#v2.8.0"
+        "superdesk-core": "github:superdesk/superdesk-client-core#release/2.8",
+        "superdesk-planning": "github:superdesk/superdesk-planning#release/2.8"
     },
     "devDependencies": {
-        "@superdesk/build-tools": "^1.0.19"
+        "@superdesk/build-tools": "^1.0.19",
+        "@types/node": "22.5.1"
```

Comment:
afaik this is not needed

Reply:
build failed the same way it did, last week or so when I was updating again

Reply:
check here - https://github.com/superdesk/superdesk-cp/actions/runs/11177624646/job/31073485297